package.json

```diff
@@ -50,7 +50,7 @@
     "docker:build:latest": "yarn docker:build:packaging && yarn docker:build:images:latest",
     "docker:build:local": "yarn docker:build:packaging && yarn docker:build:images:local",
     "docker:build:packaging": "yarn docker:scripts:build && docker build --tag api3/airnode-packaging:latest --file docker/Dockerfile .",
-    "docker:scripts:build": "ncc build docker/scripts/cli.ts -o docker/scripts/dist --no-cache --minify --source-map --transpile-only",
+    "docker:scripts:build": "ncc build docker/scripts/cli.ts -o docker/scripts/dist --no-cache --source-map --transpile-only",
```

Comment:
Btw. I am also not sure about why this should be needed... Maybe the new AWS is ESM and that causes problems??? (ESM is often to blame for stuff :D )

Reply:
I ran into this on another PR as well so I guess it was caused by some other dependency. It doesn't really matter though.